sovware/directorist
# Improve/dashboard navigation

## PR Type
What kind of change does this PR introduce?


## Description
This PR improves navigation system in dashboard. Now dashboard navigation system supports dynamic regex links.

Ex: #/page/\d+/edit.

Follow the steps bellow to test the feature:
1. Install and activate HelpGent 2 and Directorist - HelpGent Integration plugin.
2. Create a form and go to the edit page link.
3. Reload the page.
4. If the edit page remains active then the new feature is working properly.
5. Navigate to other links of the menu, reload and check if they are working too properly.

## Any linked issues
Fixes #

## Checklist
